blocks/api/serializer.js

```diff
@@ -111,6 +111,11 @@ export function serializeAttributes( attrs ) {
 
 export function serializeBlock( block ) {
 	const blockName = block.name;
+
+	if ( 'core/nextpage' === blockName ) {
+		return '<!--nextpage-->';
+	}
+
```

Comment:
Hmm, I had another look and it seems like the problem just was the missing `save()` method in the test, no other warnings. So I guess that resolved itself.